web/src/stories/Introduction.stories.mdx

```diff
@@ -0,0 +1,219 @@
+import { Meta } from "@storybook/addon-docs";
+import Code from "./assets/code-brackets.svg";
+import Colors from "./assets/colors.svg";
+import Comments from "./assets/comments.svg";
+import Direction from "./assets/direction.svg";
+import Flow from "./assets/flow.svg";
+import Plugin from "./assets/plugin.svg";
+import Repo from "./assets/repo.svg";
+import StackAlt from "./assets/stackalt.svg";
+
+<Meta title="Example/Introduction" />
```

Comment:
this page and the icons can honestly be deleted. lmk if you feel like it's unnecessary

Reply:
yeah let's get rid of it.